packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -8,16 +8,16 @@ module.exports = class SingleSpaReactGenerator extends Generator {
     super(args, opts);
 
     this.option("packageManager", {
-      type: String
+      type: String,
```

Comment:
These prettier changes are due to https://github.com/single-spa/create-single-spa/pull/110, for which I have no explanation.

Reply:
#110  is this same PR?

Reply:
lol oops copy paste error. Try https://github.com/single-spa/create-single-spa/issues/86